- Disable anonymous stats.
- Disable GPU.
- Adjust the fonts list to fonts available in base.


--- data/js/conf.js.orig	Wed Sep 19 22:30:47 2012
+++ data/js/conf.js	Wed Sep 19 22:46:46 2012
@@ -25,7 +25,7 @@
     , 'proxy_auth': false
     , 'proxy_auth_name': ''
     , 'proxy_auth_password': ''
-    , 'use_anonymous_stat': true
+    , 'use_anonymous_stat': false
     , 'shortcut_summon_hotot': '<Alt>C'
     , 'size_w': 500
     , 'size_h': 550
@@ -33,7 +33,7 @@
     , 'close_to_exit': false
     , 'sign_in_automatically': false
     , 'starts_minimized': false
-    , 'font_list': ['Arial', 'Wide', 'Narrow', 'Calibri', 'Cambria', 'Comic Sans MS', 'Consolas', 'Corsiva', 'Courier New', 'Droid Sans', 'Droid Serif', 'Syncopate', 'Times New Roman']
+    , 'font_list': ['DejaVu Sans', 'DejaVu Sans Mono', 'DejaVu Serif']
     , 'use_default_reply_all': false
     , 'pos_x': 0
     , 'pos_y': 0
@@ -55,7 +55,7 @@
         , 'font_size': 11
         , 'line_height': 1.4
         , 'enable_animation': true
-        , 'enable_gpu_acceleration': true 
+        , 'enable_gpu_acceleration': false
           // Behaviors
         , 'use_preload_conversation': true
         , 'use_alt_retweet': false
@@ -98,7 +98,7 @@
         , 'font_size': 11
         , 'line_height': 1.4
         , 'enable_animation': true
-        , 'enable_gpu_acceleration': true
+        , 'enable_gpu_acceleration': false
           // Behaviors
         , 'use_preload_conversation': true
         , 'use_alt_retweet': false
